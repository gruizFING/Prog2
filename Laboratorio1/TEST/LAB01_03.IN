conectar servidor
autenticar
enviar datos
procesar 4567 09ya tr45
desconectar
salir