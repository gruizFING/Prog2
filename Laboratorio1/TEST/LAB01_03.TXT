conectar servidor
Comando leido: conectar
Con los siguientes parametros: servidor
autenticar
Comando leido: autenticar
Sin parametros
enviar datos
Comando leido: enviar
Con los siguientes parametros: datos
procesar 4567 09ya tr45
Comando leido: procesar
Con los siguientes parametros: 4567,09ya,tr45
desconectar
Comando leido: desconectar
Sin parametros
salir
Fin!