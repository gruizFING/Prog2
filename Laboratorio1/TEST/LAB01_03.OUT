Comando leido: conectar
Con los siguientes parametros: servidor
Comando leido: autenticar
Sin parametros
Comando leido: enviar
Con los siguientes parametros: datos
Comando leido: procesar
Con los siguientes parametros: 4567,09ya,tr45
Comando leido: desconectar
Sin parametros
Fin!